dracut.sh

```diff
@@ -286,11 +286,17 @@ dropindirs_sort()
 rearrange_params()
 {
     # Workaround -i, --include taking 2 arguments
-    set -- "${@/--include/++include}"
-
-    # This prevents any long argument ending with "-i"
-    # -i, like --opt-i but I think we can just prevent that
-    set -- "${@/%-i/++include}"
+    newat=()
+    for i in "$@"; do
+        if [[ $i =~ ^-i(.*) ]]; then
+            newat+=("++include" "${BASH_REMATCH[1]}") # Replace -i by ++include
+        elif [[ $i == "--include" ]]; then
+            newat+=("++include") # Replace --include by ++include
+        else
+            newat+=("$i")
+        fi
+    done
```

Comment:
```    for i in "$@"; do
        if [[ $i == "-i" ]] || [[ $i == "--include" ]]; then
            newat+=("++include") # Replace "--include" or "-i" by "++include"
        else
            newat+=("$i")
        fi
    done
```
should be good enough. 
I don't think we should support ```dracut -ifoo bar […]```

Reply:
Fair enough. You or me?